decentraland/unity-renderer
# fix: CJK fonts are not being rendered

## What does this PR change?
When you write Chinese, Japanese or Korean symbols, they are not being rendered in the UI.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/cjk-fonts-issues
2. Try to write different phrases in chinese, japanese and korean (copy-paste from the translator for that if needed) in the chat.
3. Notice the text is being showed correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
